Objects/codeobject.c

```diff
@@ -1520,6 +1520,14 @@ PyCode_GetFreevars(PyCodeObject *code)
     return _PyCode_GetFreevars(code);
 }
 
+int
+_PyCode_GetNumFrameSlots(PyCodeObject *code)
+{
+    /* This function needs to remain in sync with Tools/build/deepfreeze.py */
+    assert(code->co_framesize >= (int)FRAME_SPECIALS_SIZE);
+    return code->co_framesize - FRAME_SPECIALS_SIZE;
```

Comment:
Odd that one line needs an `(int)` cast and the other doesn't. This actually makes me think that the cast belongs in the definition of `FRAME_SPECIALS_SIZE`. @markshannon ?

Reply:
Adding the case to `FRAME_SPECIALS_SIZE` makes sense.
It is an oddity of the C compiler that it doesn't treat `sizeof(int)` as `4` but as a `size_t` value which needs to be cast.

Reply:
I know. My question was about why apparently `int >= size_t` is a problem (which we solve with a cast) but `int - size_t` isn't. I'm guessing that the `>=` converts the `int` to `size_t` and (regardless of whether that does sign extension) negative ints will appear to be larger than zero, while `-` converts the `size_t` to `int` and all's well?